src/pages/post-purchase.tsx

```diff
@@ -0,0 +1,477 @@
+import React, {FunctionComponent} from 'react'
+import {Formik} from 'formik'
+
+import {supabase} from 'utils/supabase'
+import {useViewer} from 'context/viewer-context'
+
+//! delete these
+import useLastResource from 'hooks/use-last-resource'
+import {isEmpty} from 'lodash'
+import Link from 'next/link'
+import Image from 'next/image'
+import Spinner from 'components/spinner'
+import {IconTwitter} from 'components/share'
+import usePurchaseAndPlay from 'hooks/use-purchase-and-play'
+import {Topic} from 'types'
+//!
+
+type LoginFormProps = {
+  HeaderImageComponent?: React.FC
+  className?: string
+  button?: string
+  label?: string
+  formClassName?: string
+  onSubmit: (value: any, setIsError: any) => void
+}
+
+const PostPurchaseSurvey: FunctionComponent<any> = ({
+  question = 'What was the last thing that held you back from purchasing?',
+}) => {
+  const {viewer} = useViewer()
+
+  let onSubmit = async (values: any) => {
+    const {response} = values
+
+    await supabase.from('responses').insert({
+      email: viewer?.email,
+      question: question,
+      response: response,
+    })
+  }
+  return (
+    <div className="flex flex-col items-center p-16 my-16 bg-gray-100 rounded-lg dark:bg-gray-800">
+      <Form
+        className="flex flex-col items-center justify-center w-full mx-auto text-white"
+        label="Feedback:"
+        formClassName="max-w-xs md:max-w-sm mx-auto w-full"
+        button="Submit Feedback"
+        onSubmit={onSubmit}
+      >
+        <div className="text-center">
+          <p className="mt-4 text-base font-normal text-black dark:text-white sm:text-lg">
+            {question}
+          </p>
+        </div>
+      </Form>
+    </div>
```

Comment:
This is essentially the `OnlinePresenceEmailEntryForm` repurposed for this context